src/Pelagos/Bundle/AppBundle/Command/RebuildElasticsearchIndexCommand.php

```diff
@@ -1,83 +0,0 @@
-<?php
```

Comment:
I think its better if we keep the custom rebuild elasticsearch command because it can also index individual UDIs

Reply:
This can be done with:
bin/console fos:elastica:populate --index pelagos
bin/console fos:elastica:populate --index search_pelagos